addons/account/views/report_invoice.xml

```diff
@@ -200,7 +200,7 @@
                 <t t-foreach="payments_vals" t-as="payment_vals">
                     <tr>
                         <td>
-                            <i class="oe_form_field text-right oe_payment_label">Paid on <t t-esc="payment_vals['date']"/></i>
+                            <i class="oe_form_field text-right oe_payment_label">Paid on <t t-esc="payment_vals['date']" t-options="{&quot;widget&quot;: &quot;date&quot;}"/></i>
```

Comment:
Why not putting simple quotes instead of `&quot;` for better reading?

Reply:
I changed it to t-options='{"widget": "date"}' as suggested to keep it consistent with other t-options.